### ISSUE TITLE
Formatter overwrites my file content

### ISSUE THREAD

**Issue by awoehrl** [sqlfluff/vscode-sqlfluff#43]
Hi,

I'm using sqlfluff (0.9.1) with dbt (1.0.1). When I activate the format option in the vscode-sqlfluff extension, my model file will get overwritten by some dbt output:

```
13:18:09  [WARNING]: Deprecated functionality
The `source-paths` config has been renamed to `model-paths`. Please update your
`dbt_project.yml` configuration to reflect this change.

13:18:09  [WARNING]: Deprecated functionality
The `data-paths` config has been renamed to `seed-paths`. Please update your
`dbt_project.yml` configuration to reflect this change.

```

When using sqlfluff from the command line (lint or fix option), I don't see this behaviour.

**Comment by pblankley** [comment]
+1 for this one. When I have the form
atOnSave option enables for sqlfluff it writes this text into the file it's supposed to be formatting

<img width="360" alt="Screen Shot 2022-02-16 at 2 14 10 PM" src="https://user-images.githubusercontent.com/31221512/154349719-de178860-ef45-43fc-a030-aeca5c780db7.png">

But when I run `sqlfluff fix` from the cli it just shows me this

<img width="392" alt="Screen Shot 2022-02-16 at 2 14 48 PM" src="https://user-images.githubusercontent.com/31221512/154349836-1eecb93e-ae21-4311-a3bc-afcccfd895a2.png">

**Comment by RobertOstermann** [comment]
This should be fixed in v0.0.6
